python/spacewalk/spacewalk-backend.spec

```diff
@@ -50,10 +50,11 @@ Name:           spacewalk-backend
 Version:        5.1.5
 Release:        0
 Summary:        Common programs needed to be installed on the Spacewalk servers/proxies
-License:        GPL-2.0-only
+License:        Apache-2.0 AND GPL-2.0-only
```

Comment:
These two are not compatible i'm afraid.

Reply:
> GPL-2.0 application code

the issue is `GPL-2.0-only`

Reply:
> > GPL-2.0 application code
> 
> the issue is `GPL-2.0-only`

Isn't that just an explicit statement that it's not GPL-2.0-or-later?

Reply:
`GPL-2.0-only` means that the license  is strictly  GPL v2. But this specific one is not compatible with Apache2.0.